src/Attribute.h

```diff
@@ -0,0 +1,42 @@
+#ifndef ATTRIBUTE_H
+#define ATTRIBUTE_H
+
+#include <string>
+#include <vector>
+using namespace std;
+
+class AttributeGenerator;
+
+class Attribute
```

Comment:
Please write comments for member variables and functions. Likewise for class. I can be brief, but it would be useful for reviewers.

Reply:
Done. Please check updated changes.